NitroxServer-Subnautica/GameLogic/Entities/SimulationWhitelist.cs

```diff
@@ -70,7 +70,8 @@ public class SimulationWhitelist
             TechType.Shuttlebug.Model(),
             TechType.Bloom.Model(),
             TechType.RockPuncher.Model(),
-            TechType.Peeper.Model()
+            TechType.Peeper.Model(),
+            TechType.Jumper.Model()
```

Comment:
Good catch.  We may want to see if there were any others that I missed since my gameplay knowledge isn't very good.

Reply:
I know of one. The Precursor crawler bot but Idk the Techtype for it

Reply:
Pretty sure if you are refering to the Alien Robot its: PrecursorDroid